locales/fr-FR/messages.po

```diff
@@ -1633,7 +1633,7 @@ msgid "You are currently working offline."
 msgstr ""
 
 msgid "You are encountering a prolonged connectivity loss. "
-msgstr "Vous rencontrez actuellement une perte de connectivité prolongée."
+msgstr "Vous rencontrez une perte de connectivité prolongée. "
```

Comment:
**nitpick:** please avoid putting specific content in generic extension.
This is a customer specific flavour, hence it should go to the customer's extension